static/js/hotkey.js

```diff
@@ -272,7 +273,7 @@ exports.process_escape_key = function (e) {
         return true;
     }
 
-    hashchange.go_to_location("");
+    hashchange.go_to_location("#");
```

Comment:
Why does this line change?

Reply:
I don't remember. I can see how this change doesn't belong here. Removed.